src/Github-api.js

```diff
@@ -0,0 +1,42 @@
+const request = require("request-promise");
+
+const host = 'https://api.github.com';
+
+class Github {
+    constructor() {
```

Comment:
I'd configure your IDE asap to make sure the indentation size is 2 spaces, not 4